guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -3,31 +3,51 @@
 
 Each provisioning type requires some network configuration. Use this chapter to configure network services in {ProjectServer}'s integrated {SmartProxy}.
 
-New hosts must have access to your {SmartProxyServer}. A {SmartProxyServer} can be either {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer}. You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to the {ProjectServer} directly, or when the content is synchronized with the {SmartProxyServer}. Provisioning using the external {SmartProxyServer} can save on network bandwidth.
+New hosts must have access to your {SmartProxyServer}. {SmartProxyServer} can be either {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer}. You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to the {ProjectServer} directly, or when the content is synchronized with the {SmartProxyServer}. Provisioning using the external {SmartProxyServer} can save on network bandwidth.
 
-Configuring the {SmartProxyServer} has two basic requirements:
+Configuring {SmartProxyServer} has two basic requirements:
 
 . Configuring network services. This includes:
   * Content delivery services
   * Network services (DHCP, DNS, and TFTP)
   * Puppet configuration
 . Defining network resource data in {ProjectServer} to help configure network interfaces on new hosts.
 
-The following instructions have similar applications to configuring standalone {SmartProxyServer}s managing a specific network. To configure {Project} to use external DHCP, DNS, and TFTP services, see link:{BaseURL}installing_satellite_server_from_a_connected_network/configuring_external_services[Configuring External Services] in _Installing {ProjectServer} from a Connected Network_.
+The following instructions have similar applications to configuring standalone {SmartProxies} managing a specific network. To configure {Project} to use external DHCP, DNS, and TFTP services, see link:{BaseURL}installing_satellite_server_from_a_connected_network/configuring_external_services[Configuring External Services] in _Installing {ProjectServer} from a Connected Network_.
 
-.DHCP Ranges
-You can define the same DHCP range in {ProjectServer} for both Discovered and Provisioned systems, but use a separate range for each service within the same subnet.
+[[networking_resources]]
+.Network Resources
```

Comment:
I like how "Configuring Provisioning Resources" is a separate section. To me, that many subtitles (.Subtitle) complicate the text and make it more difficult to comprehend. I think that if we turn this into a subsection as follows, it will improve structure. Do you agree?
 == Network Resources Overview